source/contracts/factories/AuctionFactory.sol

```diff
@@ -1,17 +1,18 @@
 pragma solidity 0.4.24;
 
 
-import 'libraries/Delegator.sol';
+import 'libraries/CloneFactory.sol';
 import 'IController.sol';
+import 'IControlled.sol';
 import 'reporting/IUniverse.sol';
 import 'reporting/IAuction.sol';
 
 
-contract AuctionFactory {
+contract AuctionFactory is CloneFactory {
     function createAuction(IController _controller, IUniverse _universe) public returns (IAuction) {
-        Delegator _delegator = new Delegator(_controller, "Auction");
-        IAuction _Auction = IAuction(_delegator);
-        _Auction.initialize(_universe);
-        return _Auction;
+        IAuction _auction = IAuction(createClone(_controller.lookup("Auction")));
```

Comment:
Gas saving here! Constructing the delegator with address instead of forcing lookup on each invocation. 👍